.github/workflows/release-cli.yml

```diff
@@ -0,0 +1,24 @@
+on:
```

Comment:
You can check the result of that pipeline on the fork I created: https://github.com/mathnogueira/tracetest/releases/tag/v0.4.5

Reply:
nice way to test this kind of stuff